samples/matter/common/src/bridge/matter_bridged_device.cpp

```diff
@@ -73,6 +74,61 @@ CHIP_ERROR MatterBridgedDevice::HandleReadDescriptor(AttributeId attributeId, ui
 		uint32_t featureMap = GetDescriptorFeatureMap();
 		return CopyAttribute(&featureMap, sizeof(featureMap), buffer, maxReadLength);
 	}
+	case Clusters::Descriptor::Attributes::DeviceTypeList::Id: {
+		return CopyAttribute(mDeviceTypeList, sizeof(mDeviceTypeList), buffer, maxReadLength);
+	}
+	case Clusters::Descriptor::Attributes::ServerList::Id: {
+		return CopyAttribute(mServerIdsList, sizeof(mServerIdsList), buffer, maxReadLength);
+	}
+	case Clusters::Descriptor::Attributes::ClientList::Id: {
+		// no clients enabled
+		return CopyAttribute(nullptr, 0, buffer, maxReadLength);
```

Comment:
What do you think about creating an `mClientsList` field in the class and assigning nullptr to it? This would make the implementation more generic, as we do not support client and parts list, but user would be able to set this field value to something else without modifying the core code. The same applies to parts list.

Reply:
Ok, we can do it.